### ISSUE TITLE
wiki: CodeReviewComments: change "named returns" to "naked returns"

### ISSUE THREAD

**Issue by jalan** [golang/go#29738]
### What did you do?

I read [CodeReviewComments#named-result-parameters](https://github.com/golang/go/wiki/CodeReviewComments#named-result-parameters).

### What did you expect to see?

"Corollary: it's not worth it to name result parameters just because it enables you to use naked returns."

### What did you see instead?

"Corollary: it's not worth it to name result parameters just because it enables you to use named returns."

It should say "naked returns" here, right? Easy fix to make, just posting here to respect the rules. "Please discuss changes before editing this page, even minor ones." 😃

**Comment by mvdan** [comment]
Sounds to me like it's a human error, yes. Would like another pair of eyes on this to be sure I'm not missing anything - cc @dmitshur @bradfitz.

**Comment by ianlancetaylor** [comment]
I agree that it should be "naked returns".  Go ahead and change it.  Thanks.

(It's odd because the original source for that comment is a Google-internal doc, and in that doc it says "naked returns.")

**Comment by dominikh** [comment]
```
commit baec72a245ccc8350a5703497156484f5652b848
Author: luantao <bibicut@vip.qq.com>
Date:   Thu Jul 5 22:45:33 2018 +0800

    I think this place is wrong.

diff --git a/CodeReviewComments.md b/CodeReviewComments.md
index 32eed16..64b4100 100644
--- a/CodeReviewComments.md
+++ b/CodeReviewComments.md
@@ -458,7 +458,7 @@ func (f *Foo) Location() (lat, long float64, err error)
 
 Naked returns are okay if the function is a handful of lines. Once it's a medium
 sized function, be explicit with your return values. Corollary: it's not worth it
-to name result parameters just because it enables you to use naked returns.
+to name result parameters just because it enables you to use named returns.
 Clarity of docs is always more important than saving a line or two in your function.
 
 Finally, in some cases you need to name a result parameter in order to change
```

**Comment by ianlancetaylor** [comment]
I really really wish there were some way to get notified on each change to the wiki.

**Comment by dmitshur** [comment]
> I really really wish there were some way to get notified on each change to the wiki.

@ianlancetaylor How would you like to receive notifications? Each change to all pages, or only particular pages?

**Comment by ianlancetaylor** [comment]
@dmitshur I personally would like to see each change to all pages, just as I see each change to the any portion of the source code.
